core/routemgmt/common/apigw-utils.js

```diff
@@ -302,7 +302,7 @@ function getApis(gwInfo, spaceGuid, bpOrApiName) {
   }
   var options = {
     followAllRedirects: true,
-    url: gwInfo.gwUrl+'/'+encodeURIComponent(spaceGuid)+'/apis',
+    url: gwInfo.gwUrl+'/'+encodeURIComponent(spaceGuid)+'/apis?limit='+limit+'&skip='+skip,
```

Comment:
alas, the `limit` and `skip` query parameters are supported by the openwhisk backend/controller, but not the api gateway
https://github.com/apache/incubator-openwhisk-apigateway/blob/master/scripts/lua/management/routes/apis.lua

Reply:
@mdeuser There's a PR for that too: https://github.com/apache/incubator-openwhisk-apigateway/pull/335, but it's blocked until the build problem is solved.